app/models/manageiq/providers/cloud_automation_manager/configuration_manager/refresher.rb

```diff
@@ -4,8 +4,16 @@ module CloudAutomationManager
     class ConfigurationManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       def parse_legacy_inventory(manager)
         manager.with_provider_connection do |connection|
-          data = collect_configuration_inventory(connection, manager.url)
+          # get templates
+          template_url_str = ":30000/cam/api/v1/templates"
+          type = "template"
+          data = collect_configuration_inventory(connection, manager.url, template_url_str, type)
           ConfigurationManager::RefreshParser.configuration_inv_to_hashes(data)
+          # get configured systems
+          stack_url_str = ":30000/cam/api/v1/stacks"
+          type = "stacks"
+          data_systems = collect_configuration_inventory(connection, manager.url, stack_url_str, type)
+          ConfigurationManager::RefreshParser.configuration_inv_to_hashes(data_systems)
```

Comment:
And we should only be calling `configuration_inv_to_hashes` once with all of the collected data